deploy/clowdapp.yaml

```diff
@@ -549,6 +549,29 @@ objects:
         WHERE rule_results.result IN ('fail', 'error', 'unknown', 'fixed')
         GROUP BY "rules"."ref_id", "accounts"."org_id";
 
+- apiVersion: metrics.console.redhat.com/v1alpha1
+  kind: FloorPlan
+  metadata:
+    name: compliance-hms
+  spec:
+    database:
+      secretName: ${FLOORIST_DB_SECRET_NAME}
+    objectStore:
+      secretName: ${FLOORIST_HMS_BUCKET_SECRET_NAME}
+    logLevel: ${FLOORIST_LOGLEVEL}
+    suspend: ${{FLOORIST_SUSPEND}}
+    queries:
+    - prefix: hms_analytics/compliance/tests
+      query: >-
```

Comment:
Please add some `chunksize`, otherwise it defaults to 1000. I recommend to start with 200k.